src/test/java/com/thindeck/dynamo/ClasspathRule.java

```diff
@@ -97,6 +97,7 @@ public boolean apply(final Class<?> input) {
                         && !name.endsWith("ITCase")
                         && !name.endsWith("ClasspathRule")
                         && !name.endsWith("RepoRule")
+                        && !name.endsWith("DyTxn")
```

Comment:
@oruam85 do we need this exclude now that we have removed the `@Immutable` annotation?